full-ingestion/src/extensions/algolia-example/mappers/product.mapper.js

```diff
@@ -3,7 +3,7 @@ export default function map(product, locale) {
   let mappedProduct = {};
   let categories = product.categories.map((category) => {
     return {
-      id: category.id,
+      id: category.obj?.key
```

Comment:
Not clear on this- why key is being assigned to id?

Reply:
It was a specific requirement from the Starters team, They had this problem using Ids(Since they changed the CT project or they were using 2 CT projects).
If they get key values, It will be the same in both the projects.

Reply:
This will cause confusion in customers mind using this connector although approving to unblock since CoFe implementation might be already using key as id in frontend. This is a techdebt for CoFe to resolve.